functors task (1)/student/include/impl.h

```diff
@@ -16,16 +47,31 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand brand)
 {
+  switch(brand){
```

Comment:
please never use switches in such way :)

if your return value is duplicated, you can use fallthrough.
But, for these two types it is enough to use comparing like:

return brand == BeerBrand::Corona || brand == BeerBrand::HoeGaarden